src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-app-launch-time-android-apps.mdx

```diff
@@ -35,7 +36,7 @@ Your app launch time is reported with the `AppLaunch/Cold` and `AppLaunch/Hot` [
 Here's a sample query for capturing cold and hot launch times: 
 
 ```sql
-  SELECT average(newrelic.timeslice.value ) AS `AppLaunch/Cold` FROM Metric WHERE metricTimesliceName = ‘AppLaunch/Cold’ AND `entity.guid` = ‘your app token here’ SINCE 1664218800000 UNTIL 1664220600000 TIMESERIES
+  SELECT average(newrelic.timeslice.value ) AS 'AppLaunch/Cold' FROM Metric WHERE metricTimesliceName = 'AppLaunch/Cold' AND entity.guid = 'your app token here' SINCE 1664218800000 UNTIL 1664220600000 TIMESERIES
```

Comment:
It doesn't look like the "your app token here" doesn't align with our style guide about customer-specific placeholders: https://docs.newrelic.com/docs/style-guide/formatting/code-examples/#value-placeholders